sktime/transformations/series/fourier.py

```diff
@@ -152,6 +165,15 @@ def _fit(self, X, y=None):
         # store the integer form of the minimum date in the prediod index
```

Comment:
move the comment to be just above the line:
`self.min_t_ = np.min(X.index.astype(int))`